sherlock_project/resources/data.json

```diff
@@ -2802,5 +2798,12 @@
     "url": "https://www.znanylekarz.pl/{}",
     "urlMain": "https://znanylekarz.pl",
     "username_claimed": "janusz-nowak"
+  },
+  "Bluesky": {
+    "errorMsg": "<title>@{} — Bluesky</title>",
+    "errorType": "message",
+    "url": "https://bsky.app/profile/{}.bsky.social",
```

Comment:
I'm not sure this is going to work super well. If the user doesn't have a display name, I think the title of the page will match this even if the user exists. It might be better to use the `status_code` errorType and then specify the actual API call that looks up the user. That should respond with a 400 if the user doesn't. The API call would be `https://public.api.bsky.app/xrpc/com.atproto.identity.resolveHandle?handle={}.bsky.social`

Reply:
Hey @ibnaleem before I run through things do you have any thoughts here?